### PR TITLE
Serenity Updates - October - Reorder articles + new description

### PR DESCRIPTION
**Pull request by LaureBro**
Reorder articles from 1 to 6, adding new monthly description

### CODE REVIEW

File: content/updates/index.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   },
   "2020-10": {
     "title": "🎃 October Serenity updates",
-    "description": "New Serenity updates of October",
+    "description": "This fall, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid improvements, better prioritize your data quality focus areas.",
```
Comment (NolwennP) [inline content/updates/index.json:9]: Your description is a bit too short.

Comment (LaureBro) [reply content/updates/index.json:9]:
```suggestion
    "description": "This fall, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid enhancements, better prioritize your data quality focus areas and benefit from other improvements.",
```

File: content/updates/index.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   },
   "2020-10": {
     "title": "🎃 October Serenity updates",
-    "description": "New Serenity updates of October",
+    "description": "This fall, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid enhancements, better prioritize your data quality focus areas and benefit from other improvements.",
```
Comment (NolwennP) [inline content/updates/index.json:9]: Still too short @LaureBro. I suggest adding:
```suggestion
    "description": "This fall, in October, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid enhancements, better prioritize your data quality focus areas and benefit from other improvements.",
```

Comment (LaureBro) [reply content/updates/index.json:9]:
```suggestion
    "description": "The month of October rhymes with better productivity, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid enhancements, better prioritize your data quality focus areas and benefit from other improvements.",
```

Comment (NolwennP) [reply content/updates/index.json:9]:
```suggestion
    "description": "The month of October rhymes with better productivity, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid, better prioritize your data quality focus areas and benefit from other improvements.,
```

Comment (LaureBro) [reply content/updates/index.json:9]:
```suggestion
    "description": "October rhymes with better productivity, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid enhancements, better prioritize your data quality focus areas and benefit from other improvements.",
```

**Review by NolwennP — APPROVED** [review]

**Pull request merged**